utils/elastic_search/elastic_search_utils.py

```diff
@@ -0,0 +1,111 @@
+from typing import List
+
+from common.configuration import ELASTIC_INDEX, ELASTIC_RESULTS_PER_PAGE, ELASTIC_SEARCH_TIMEOUT
+
+
+class ElasticSearchUtils:
+    def build_search_query(self, search_text: str, sort_by: str, order: str, search_after: List[str]) -> dict:
+        """Build the search query that will be sent to ElasticSearch given the parameters provided by API Client.
+        Access this link for more details on how ES search API works:
+        https://github.com/elastic/elasticsearch-py/blob/main/elasticsearch/_sync/client/__init__.py#L3335
+
+        :param search_text: Input text requested by user
+        :type search_text: str
+
+        :param sort_by: How the data will be sorted (By ID, name, or symbol)
+        :type sort_by: str
+
+        :param order: If the order of the sorted data will be asc/desc
+        :type order: str
+
+        :param search_after: A list with two entries with information of the next page
+        :type search_after: List[str]
+        """
+
+        # Default sort order, if nothing is passed
+        sort_order = ['id', 'name', 'symbol']
+
+        if not sort_by or sort_by == 'uid':
+            sort_by = 'id'
+
+        sort_fields_data_type = {
+            'id': 'keyword',
+            'name': 'keyword',
+            'symbol': 'keyword',
+            'nft': 'keyword'
```

Comment:
Can we sort by `nft`?

Reply:
Not now. I will remove this column. We will need to change a little bit how this data is stored on ElasticSearch in order to enable sorting by this field.